apps/indexer/src/api/index.ts

```diff
@@ -43,6 +48,17 @@ app.onError(errorHandler);
 app.use("/", graphql({ db, schema }));
 app.use("/graphql", graphql({ db, schema }));
 
```

Comment:
Missing null check for governorClient. If getGovernor returns null/undefined, this will cause a runtime error when trying to instantiate ProposalsService.